### PR TITLE
Add parameter to customize the number of manilaAPI processes

### PR DESCRIPTION
**Pull request by fmount**

### CODE REVIEW

File: config/crd/bases/manila.openstack.org_manilas.yaml [hunk 1]
```diff
@@ -672,6 +672,11 @@ spec:
                       caBundleSecretName:
                         type: string
                     type: object
+                  workers:
+                    default: 3
```
Comment (fmount) [inline config/crd/bases/manila.openstack.org_manilas.yaml:676]: @gouthampacha is this a good "default" ? Do we want to stay w/ `4` (the original hardcoded value), or do we want something else?

**Comment by stuggi** [comment]
I am against adding a parameter for something you can use the custom config way to set it. we do not want to introduce tht params

**Comment by stuggi** [comment]
> I am against adding a parameter for something you can use the custom config way to set it. we do not want to introduce tht params

ok, wait this is different. this is for httpd customization

**Comment by stuggi** [comment]
form httpd tuning, check this keystone PR https://github.com/openstack-k8s-operators/keystone-operator/pull/500 . we should do the same here

**Pull request closed without merging**